packages/manager/modules/pci/src/projects/project/data-processing/terminate-job/terminate-job.controller.js

```diff
@@ -0,0 +1,30 @@
+export default class {
+  /* @ngInject */
+  constructor($state, dataProcessingService) {
+    this.$state = $state;
+    this.dataProcessingService = dataProcessingService;
+    this.deleteJob = this.deleteJob.bind(this);
+  }
+
+  deleteJob() {
+    this.dataProcessingService
+      .terminateJob(this.projectId, this.jobId)
+      .then(() => {
+        this.$state.go(
```

Comment:
Always return your `Promise`s please
```suggestion
        return this.$state.go(
```